ansible_builder/utils.py

```diff
@@ -1,8 +1,56 @@
 import subprocess
+import sys
+import os
 
+from . import constants
```

Comment:
A design tweak I would like to make is avoiding importing other modules inside of utils or constants. Given how we're moving actual source around, I would prefer hard code the value twice.

To go even further, the default that would make the most sense is to search _all_ the paths in the Ansible core default, and return categories for each in the YAML output, exactly like `ansible-galaxy collection list`. That way, the method speaks a universal truth and cuts off ties to implementation of the rest of the project.